packages/contracts/src/dump.ts

```diff
@@ -0,0 +1,131 @@
+/* External Imports */
+import * as path from 'path'
+import { ethers } from 'ethers'
+import * as Ganache from 'ganache-core'
+import { deployAllContracts, RollupDeployConfig } from './deployment'
+
+interface StorageDump {
+  [key: string]: string
+}
+
+export interface StateDump {
+  contracts: {
+    ovmExecutionManager: string
+    ovmStateManager: string
+  }
+  accounts: {
+    [address: string]: {
+      balance: number
+      nonce: number
```

Comment:
It would be nice to have `codeHash` as well, but it could be derived from the contract code. Removes the dependency of the consumer needing to have a `keccak` implementation.

Reply:
Good point! Will do.